### PR TITLE
[Merged by Bors] - feat: add new env names for kl-retriever (NLU-825)

### PR DESCRIPTION
**Pull request by bhuvana-ak**
**Fixes or implements NLU-825**

### Brief description. What is this change?
Add new cloud env names in general-runtime to access kl-retriever.

### Implementation details. How do you make this change?
New Types added for new envs.

### CODE REVIEW

File: lib/services/runtime/handlers/utils/knowledgeBase/types.ts [hunk 1]
```diff
@@ -1,4 +1,7 @@
 export enum CloudEnv {
   Public = 'public',
   USBank = 'usbank',
+  JPMC = 'jpmc',
+  CISCO = 'cisco',
```
Comment (pmvrmc) [inline lib/services/runtime/handlers/utils/knowledgeBase/types.ts:5]: why add these ones if they are not used?

Comment (bhuvana-ak) [reply lib/services/runtime/handlers/utils/knowledgeBase/types.ts:5]: These 2 envs will be enabled after testing the functionality in vf-test.
Soon, I'll have a follow up pr for that..

Comment (pmvrmc) [reply lib/services/runtime/handlers/utils/knowledgeBase/types.ts:5]: i though so! totally fine, but ideally they would be added when they are needed!

lets get it merged 👍

**Pull request closed without merging**